<launch>
  <arg name="parent_frame"  default="/map"/>
  <arg name="child_frame" default="/camera_link"/>
  <arg name="pose_name" default="/cur_camera_pose"/> <!--rostopic type : geometry_msgs::PoseStamped-->
  <!-- <arg name="rate" default="1"/> -->

  <node name="establishtf_Node" pkg="establishtf"  type="establishtf_Node" 
        args = "$(arg parent_frame) $(arg child_frame) $(arg pose_name) " output="screen"/>

  <arg name="focal_len"  default="1.0"/>
  <arg name="field_size" default="0.25"/>
  <arg name="resolution" default="0.1"/>
  <arg name="cloud_name" default="/cur_kp_cloud"/>  <!--rostopic type : sensor_msgs::PointCloud-->

  <node name="explore_Node" pkg="explore"  type="explore_Node" 
        args = "$(arg focal_len) $(arg field_size) $(arg resolution) $(arg cloud_name) $(arg pose_name)  " output="screen"/>

</launch>